src/app/core/components/archive-settings-dialog/archive-settings-dialog.component.ts

```diff
@@ -6,12 +6,15 @@ import { TagsService } from '@core/services/tags/tags.service';
 import { TagVO } from '@models/tag-vo';
 import { ArchiveVO } from '@models/index';
 
-type ArchiveSettingsDialogTab = 'manage-tags' | 'public-settings';
+type ArchiveSettingsDialogTab =
+  | 'manage-tags'
```

Comment:
If it's easy to change this `manage-tags` to `manage-keywords` as part of this PR, this is probably a good time for it.